src/inmanta/deploy/scheduler.py

```diff
@@ -744,6 +834,7 @@ def get_types_for_agent(self, agent: str) -> Collection[ResourceType]:
         return list(self._state.types_per_agent[agent])
 
     async def send_in_progress(self, action_id: UUID, resource_id: ResourceVersionIdStr) -> None:
+        self._timer_manager.remove_resource(Id.parse_id(resource_id).resource_str())
```

Comment:
I would put this in 'get_resource_intent_for_deploy' .

`get_resource_intent_for_deploy` is the deploy start of the scheduler, this one is the external update to the DB

Reply:
Moved to `get_resource_intent_for_deploy`